app/decorators/service_provider_session_decorator.rb

```diff
@@ -113,6 +113,10 @@ def url_options
     end
   end
 
+  def irs_attempts_api_session_id
+    @irs_attempts_api_session_id ||= request_params['irs_attempts_api_session_id']
```

Comment:
I'm not in love with keeping `irs_` in the external facing API key but I know we had that discussion already